documentation/modules/creating-migration-quick-plan-existing-source.adoc

```diff
@@ -22,7 +24,28 @@ The *Create migration plan* pane opens. It displays the source provider's name a
 . Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
 . Click *Create migration plan*.
 +
-{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan.
+{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan
+
+. VMware source providers only (Both optional):
+
+.. Preserving the CPU type of VMs that are migrated from {rhv-short}: By default, {project-short} sets the CPU type (CPU model and any flags) of migrated VMs if the type is not set specifically for the VM. This can be an issue for migrations from {rhv-short} because the CPU type of {rhv-short} VMs can be set at the cluster level, in which case {project-short} might change the type.
```

Comment:
If the VM isn't set with anything, MTV sets it as written here.
With the latest change, if the VM is set with custom CPU model, we will set it as the VM configuration set in the destination.
If you wish to use RHV's cluster CPU you will need to set `Preserve CPU model`.
https://github.com/kubev2v/forklift/pull/686 and https://github.com/kubev2v/forklift/pull/688

Reply:
Rewritten